### ISSUE TITLE
Duplicate ref detected from the same file (using two different paths)

### ISSUE THREAD

**Issue by brabalan** [org-roam/org-roam#1291]
### Description

I keep getting this error:
```
Error (org-roam): Duplicate ref //www.arte.tv/fr/videos/077342-000-A/selfie-avoir-16-ans-a-naples/ in:

A: /home/schmitta/Documents/Org/notes/20201114225606-selfie_avoir_16_ans_a_naples.org
B: 20201114225606-selfie_avoir_16_ans_a_naples.org
```

The problem is that the files A and B are the same one, the first one using the absolute path, and the second use using the relative path. How can I convince org-roam it is only one file (hence only one ref)?

### Environment

- Emacs: GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.22, cairo version 1.17.3)
 of 2020-08-28
- Framework: Doom
- Org: Org mode version 9.5 (nil @ /home/schmitta/.emacs.d/.local/straight/build-27.1/org-mode/)
- Org-roam: 1.2.3
- Org-roam commit: https://github.com/org-roam/org-roam/commit/8401784

**Comment by brabalan** [comment]
Thanks to https://org-roam.discourse.group/t/duplicate-refs-for-the-same-file-how-can-i-rebuild-the-cache/974/2 I found a workaround: `org-roam-db-clear` followed by `org-roam-db-build-cache`. I'm trying to see if I can reproduce this. One thing of note is that it happens when I shorten the title, so this might be related to #1303.

**Comment by jethrokuan** [comment]
This was fixed with #1308.

**Comment by brabalan** [comment]
Thank you!
